modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannels.testcase

```diff
@@ -146,7 +146,68 @@ definition {
 	test CreateNewChannelCompileAll {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6102 CreateNewChannelCompileAll
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Store Management",
+			portlet = "Channels");
+
+		CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+
+		Type(
+			locator1 = "TextInput#NAME",
+			value1 = "Minium Portal 2");
+
+		Select(
+			key_title = "Currency",
+			locator1 = "CommerceEntry#CATALOGS_FORM",
+			value1 = "Euro");
+
+		Select(
+			key_title = "Commerce Site Type",
+			locator1 = "CommerceEntry#CATALOGS_FORM",
+			value1 = "B2X");
+
+		Select(
+			key_title = "Buyer Order Approval Workflow",
+			locator1 = "CommerceEntry#CATALOGS_FORM",
+			value1 = "Single Approver (Version 1)");
+
+		Select(
+			key_title = "Seller Order Acceptance Workflow",
+			locator1 = "CommerceEntry#CATALOGS_FORM",
+			value1 = "Single Approver (Version 1)");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Purchase Order Number",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Guest Checkout",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Type(
+			locator1 = "TextInput#NUMBER",
+			value1 = "1");
+
+		Type(
+			locator1 = "CommerceEntry#CHANGE_TAX_CATEGORY",
+			value1 = "Normal Product");
+		Pause(locator1 = "3000");
+
+		Select(
+			key_title = "Price Type",
+			locator1 = "CommerceEntry#CATALOGS_FORM",
+			value1 = "Gross Price");
+
+		Select(
+			key_title = "Discounts Target Price Type",
+			locator1 = "CommerceEntry#CATALOGS_FORM",
+			value1 = "Gross Price");
+
+		Button.clickSave();
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
```

Comment:
The part where it is checked if all changes have been saved is missing we have only the check if the green alert is present, the step 15 of the ticket says: "Verify if the green alert appears and if all the changed are correctly saved"